File: pr1-java/networking/src/main/java/pr1Java/networking/dataTransfer/SeatsSoldDto.java
package pr1Java.networking.dataTransfer;

import java.io.Serializable;

public class SeatsSoldDto implements Serializable {

    private final Integer gameId;
    private final Integer seatsCount;

    public SeatsSoldDto(Integer gameId, Integer seatsCount) {
        this.gameId = gameId;
        this.seatsCount = seatsCount;
    }

    public Integer getGameId() {
        return gameId;
    }

    public Integer getSeatsCount() {
        return seatsCount;
    }

    public String toString() {
        return "SeatsSoldDto[" + getGameId() + " " + getSeatsCount() + "]";
    }
}

File: pr1-cs/Networking/Reflection/DataTransfer/SeatsSoldDto.cs
using System;

namespace Networking.Reflection.DataTransfer
{
    [Serializable]
    public class SeatsSoldDto
    {
        public SeatsSoldDto(int gameId, int seatsCount)
        {
            GameId = gameId;
            SeatsCount = seatsCount;
        }

        public int GameId { get; }
        public int SeatsCount { get; }

        public override string ToString()
        {
            return "SeatsSoldDto[" + GameId + " " + SeatsCount + "]";
        }
    }
}